src/smart-components/Conversation/dux/reducers.js

```diff
@@ -214,7 +214,11 @@ export default function reducer(state, action) {
       };
     }
     case actionTypes.ON_MESSAGE_UPDATED: {
```

Comment:
Add test for these two changes

Reply:
There is already a test for `ON_MESSAGE_UPDATED` event.
Let me add new tests for `ON_MESSAGE_THREAD_INFO_UPDATED` event.